src/modules/module_28503.c

```diff
@@ -57,7 +57,7 @@ u32 module_pw_min (MAYBE_UNUSED const hashconfig_t *hashconfig, MAYBE_UNUSED con
 
 static u32 polymod_checksum (const u8 *data, const u32 data_len)
 {
-  const u32 CONST[5] = { 0x3b6a57b2, 0x26508e6d, 0x1ea119fa, 0x3d4233dd, 0x2a1462b3 };
+  const u32 consts[5] = { 0x3b6a57b2, 0x26508e6d, 0x1ea119fa, 0x3d4233dd, 0x2a1462b3 };
```

Comment:
```suggestion
  const u32 CONSTS[5] = { 0x3b6a57b2, 0x26508e6d, 0x1ea119fa, 0x3d4233dd, 0x2a1462b3 };
```

Tiny issue but maybe you should leave uppercase names?

Reply:
sure, it will be better (iv seen code - it seems to coded this style ofter)